proto/grp.proto

```diff
@@ -61,7 +72,60 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
+  context.RequestContext request_context = 1;
+
+  // The ID of the group to get the users from.
+  string group_id = 2;
+
+  // Membership status filter (optional).
+  GroupMembershipStatus group_membership_status = 3;
+}
+
+message GroupUser {
```

Comment:
minor nit but if you expect this to be used primarily in the context of GetGroupUserUsers you can just nest it in that message and then people will be less likely to use it elsewhere for ... nefarious purposes :P